.openpublishing.redirection.json

```diff
@@ -906,10 +906,258 @@
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/errors-warning"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/nowin32manifest-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/resources"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/nullable-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/language"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/optimize-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/code-generation"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/nostdlib-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/advanced"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/out-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/optimizations"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/pathmap-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/advanced"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/pdb-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/advanced"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/platform-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/optimizations"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/preferreduilang-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/advanced"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/publicsign-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/security"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/recurse-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/index"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/reference-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/inputs"
+        },
+        {
+            "source_path": "docs/csharp/language-reference/compiler-options/refout-compiler-option.md",
+            "redirect_url": "/dotnet/csharp/language-reference/compiler-options/optimizations"
```

Comment:
@BillWagner This doesn't seem to be a valid redirect target. Where should this redirect actually go?

Reply:
I have a PR to fix it.